elections/kubeflow-steering-committee-elections-2023.md

```diff
@@ -1 +1,92 @@
-This file will contain guidelines, overview, timeline, phases, list of eligible voters, list of eligible candidates and results from the election.
+# Kubeflow Project Steering Committee Elections 2023 #
+## Guidelines, Timelines and Phases ##
+
+The Kubeflow Project is holding its first election for the Kubeflow Project Steering Committee (PSC).  The PSC will also be responsible for helping to setting up of Technical Oversight Committee (TOC) that is responsible for defining and maintaining the technical vision of the Kubeflow Project and an Administrative Oversight Committee (AOC) that is responsible for defining and maintaining the outreach, growth and other non-code specific items of the Kubeflow project. The TOC and AOC will be set up at a future date and will have its own defining role in the Kubeflow project.  
```

Comment:
Can you change everywhere from "PSC" to "KSC"? The latter was used in governance related documents so far.

Reply:
I am in favor of the 3 or fewer Committees i.e. KSC, TOC and perhaps one more.     I am concern about admin overhead, overlapping roles and budgets.

Reply:
@james-jwu  - yes, I'll change that.

@jbottum - it's really just 3 as the TOC and AOC (which is what you were saying as communications) so it's not a change from what you originally proposed.

Reply:
Original proposal is to have 3 committees: https://github.com/kubeflow/community/blob/master/proposals/GOVERNANCE.md: KSC, TOC and KCC

For AOC do we need a committee, which requires formal process for motions and voting? Can the AOC be an executive group instead of a committee?
 The KSC charter covers much of AOC responsibilities: https://github.com/kubeflow/community/pull/650/files

Reply:
Same question - What are the responsibilities of AOC?   Is there an overlap?